config/secrets.yml

```diff
@@ -1,8 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rails secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
 development:
   secret_key_base: 'd3bdf173765bfe70566dd6a6bd9068b76c064af321955a6fc7d775bb383efcba1dbfe825dfc999f098fc133621b14ed573d9481be9aadc0321b4c6c7a5e087dd'
 
 test:
   secret_key_base: 'd3bdf173765bfe70566dd6a6bd9068b76c064af321955a6fc7d775bb383efcba1dbfe825dfc999f098fc133621b14ed573d9481be9aadc0321b4c6c7a5e087dd'
 
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
 production:
-  secret_key_base: '5e50ff60abaf45e9fc168cf7d6bd8d0fa13b1d7998839659bf8fb1ecbb3e564c7fea9c38d93168679ec54987d0f08e261064684000a4da193fde41f0ac110f1d'
```

Comment:
This was not so secret in the past. In any case it's OK to keep in git history since we're not really using it.